README.md

```diff
@@ -32,31 +32,418 @@ There's a very unstable version of the plugin available to try out. It's held to
 
 `yarn rw dev`
 
-## API Reference
+> **Note**
+>
+> The following steps won't be needed for much longer
```

Comment:
Recommend you get rid of this as it's just describing how to use Stripe products instead of this plugin.